ontrack-repository-impl/src/main/resources/META-INF/db/update.41.sql

```diff
@@ -1,3 +1,3 @@
 -- 41. Increase property search key max length (#639)
 
-ALTER TABLE PROPERTIES ALTER COLUMN SEARCHKEY VARCHAR(600) NOT NULL;
+ALTER TABLE PROPERTIES ALTER COLUMN SEARCHKEY VARCHAR(600) NULL;
```

Comment:
Normally, database update files must not be updated, but DBInit is tolerant in this regard :)

In FlyWay (version 3.x), this would not work.